src/commands/lambda/tags.ts

```diff
@@ -0,0 +1,43 @@
+import {Lambda} from 'aws-sdk'
+import {TAG_VERSION_NAME} from './constants'
+import {version} from '../../../package.json'
```

Comment:
this might modify the entry point of the script. Will test locally to verify.

Reply:
I'd use something like `const { version } = require(path.join(__dirname, '../../../package.json'));` or something with `require.resolve()`.